rclpy/rclpy/node.py

```diff
@@ -1460,6 +1460,24 @@ def get_service_names_and_types_by_node(
             return _rclpy.rclpy_get_service_names_and_types_by_node(
                 capsule, node_name, node_namespace)
 
+    def get_client_names_and_types_by_node(
+        self,
+        node_name: str,
+        node_namespace: str
+    ) -> List[Tuple[str, List[str]]]:
+        """
+        Get a list of discovered service client topics for a remote node.
```

Comment:
I would update the docstring for `get_service_names_and_types_by_node` to clarify that it is for "service servers".